packages/web-components/src/components/rux-modal/rux-modal.tsx

```diff
@@ -73,26 +92,27 @@ export class RuxModal {
     handleClick(ev: MouseEvent) {
         const wrapper = this._getWrapper()
         if (ev.composedPath()[0] === wrapper) {
-            this.ruxModalClosed.emit(false)
+            // this.ruxModalClosed.emit(false)
             this.open = false
         }
     }
 
     @Watch('open')
-    validateName(isOpen: boolean) {
-        if (isOpen) {
+    handleOpen(isOpen: boolean) {
+        //? This is preventing the default focusing of a button if the footer slot exists.
+        //? I thought about runninng it anyway, but it could lead to cases where an unintentional button gets focused.
+        if (isOpen && !this.hasFooter) {
```

Comment:
I didn't want the auto-focusing running if there was a footer slot present, since it could focus a button the user doesn't  want focused.